Makefile

```diff
@@ -16,23 +16,19 @@ RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
 # Python checks
-PYTHON?=python3
+UV?=uv
 
 # installed?
-ifeq (, $(shell which $(PYTHON) ))
-  $(error "PYTHON=$(PYTHON) not found in $(PATH)")
+ifeq (, $(shell which $(UV) ))
+  $(error "UV=$(UV) not found in $(PATH)")
 endif
```

Comment:
https://github.com/plone/cookieplone?tab=readme-ov-file#development-requirements needs to be updated to mention uv